jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/GracefulHandler.java

```diff
@@ -28,14 +28,14 @@
 /**
  * Handler to track active requests and allow them to gracefully complete.
  */
-public class GracefulShutdownHandler extends Handler.Wrapper implements Graceful
```

Comment:
Do we have a module and xml to add this to a jetty-home?  If not, we probably need one, if so, then it needs to have the rename.

Reply:
Not that I can find.

Let me add a module to this PR then.

Reply:
Done.